scss/atoms/_animations.scss

```diff
@@ -16,3 +16,17 @@
 		transform: scale(1);
 	}
 }
+
+@keyframes "loading-fade" {
```

Comment:
Since this animation ends up in global scope and doesn't have `jetpack-` prefix, it might end up overriding some theme or core animations.

Reply:
That's a good point, @simison. I'll rename the animation shortly.

Reply:
@simison This is addressed now in https://github.com/Automattic/jetpack/pull/13562. Please have a look if you get a chance.